ci/docker/buildenv/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+
+: "${REPO:=https://github.com/verilator/verilator}"
+: "${REV:=master}"
+: "${CC:=gcc}"
+: "${CXX:=g++}"
+
+GCCVERSION=`${CC} --version | grep gcc | awk '{print $4}'`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use $(...) notation instead of legacy backticked `...`.](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4756540)